web-ui/src/components/expand-more/ExpandMore.jsx

```diff
@@ -5,7 +5,7 @@ import { IconButton } from '@mui/material';
 const ExpandMore = styled(props => {
   const { expand, ...other } = props;
   return (
-    <IconButton {...other}>
+    <IconButton classes={{ root: 'expand-more' }} {...other}>
```

Comment:
I added a CSS class to this so I can apply CSS properties in pages were it is used.